lib/ansible/modules/commands/command.py

```diff
@@ -58,6 +58,12 @@
     version_added: "2.4"
     description:
       - Set the stdin of the command directly to the specified value.
+  binary_stdin:
```

Comment:
`binary_stdin` is a bit of a misnomer (run_command's fault here) as all it controls is whether to add a newline or not. I would change this option to be something like `raw_stdin` (up for debate) instead.

Reply:
cool let's change it to raw_stdin, I had no disagreements from other core members. We can always change it before 2.8 is released if someone really objects.

Reply:
renamed.  Not sure when I'll have a chance to look at tests...